app/posts/apply-for-teacher-training/2022-02-28-referee-flow-update.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Updating the referee flow
+description: Why we made made changes to the reference flow
+date: 2022-02-28
+---
+
+We made some changes to the way that references are given on Apply, after noticing that we were receiving a lot of support requests relating to references.
+
+Several people got in touch to say that they had accidentally refused to give a candidate a reference. Previously there were two links in the email to potential referees, one which led to a form collecting a reference, and one which led to a form where they could confirm that they were unable to give a reference.
+
+Even though declining a reference required someone to first click a confirmation button, our hypothesis was that people were scanning the email, clicking the second link, and then clicking the confirmation button without full reading the content.
+
+To resolve this, we updated the email so that it only contains a single link, and that people are asked whether or not they can give a reference as an explicit question, which then still required a confirmation button.
```

Comment:
```suggestion
To resolve this, we updated the email and form so that:
* there is only one link in the email
* the reference form asks whether you can give a reference
```